lms/djangoapps/learner_dashboard/programs.py

```diff
@@ -285,3 +288,8 @@ def render_iframe(self) -> str:
             )
         )
         return fragment.content
+
+
+class ProgramLiveLTI(ProgramDiscussionLTI):
```

Comment:
I'd suggest to make `get_configuration` an abstract method and move it to base class `ProgramLTI` and implement it in both `ProgramLiveLTI` and `ProgramDiscussionLTI`